src/scribe_data/extract_transform/English/verbs/query_verbs.sparql

```diff
@@ -1,28 +1,53 @@
 # All English (Q1860) verbs and the currently implemented tenses for each.
 # Enter this query at https://query.wikidata.org/.
 
-SELECT DISTINCT ?verb ?simplePast ?pastParticiple WHERE {
+SELECT DISTINCT ?infinitive ?presFPS ?presTPS ?simpPast ?pastPart WHERE {
+  # Infinitive (required)
   ?lexeme a ontolex:LexicalEntry ;
     dct:language wd:Q1860 ;
     wikibase:lexicalCategory wd:Q24905 ;
-    wikibase:lemma ?lemma .
+    wikibase:lemma ?infinitive .
+
+  # Simple Present
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?presFPSForm .
+    ?presFPSForm ontolex:representation ?presFPS ;
+      wikibase:grammaticalFeature wd:Q3910936 ;
+      FILTER NOT EXISTS { ?presFPSForm wikibase:grammaticalFeature wd:Q51929074 . }
+      FILTER NOT EXISTS { ?presFPSForm wdt:P6191 wd:Q181970 . }
+      FILTER NOT EXISTS { ?presFPSForm wikibase:grammaticalFeature wd:Q126473 . }
+  } .
+
+  # Third-person Singular
+  OPTIONAL {
+    ?lexeme ontolex:lexicalForm ?presTPSForm .
+    ?presTPSForm ontolex:representation ?presTPS ;
+    wikibase:grammaticalFeature wd:Q51929074 ;
+    wikibase:grammaticalFeature wd:Q110786 ;
+    wikibase:grammaticalFeature wd:Q3910936 ;
+    FILTER NOT EXISTS { ?presFPSForm wdt:P6191 wd:Q181970 . }
+    FILTER NOT EXISTS { ?presFPSForm wikibase:grammaticalFeature wd:Q126473 . }
+  } .
 
   # Simple Past
   OPTIONAL {
-    ?lexeme ontolex:lexicalForm ?simplePastForm .
-    ?simplePastForm ontolex:representation ?simplePast ;
-      wikibase:grammaticalFeature wd:Q1392475 ;
+    ?lexeme ontolex:lexicalForm ?simpPastForm .
+    ?simpPastForm ontolex:representation ?simpPast ;
+    wikibase:grammaticalFeature wd:Q1392475 ;
+    FILTER NOT EXISTS { ?presFPSForm wdt:P6191 wd:Q181970 . }
+    FILTER NOT EXISTS { ?presFPSForm wikibase:grammaticalFeature wd:Q126473 . }
   } .
 
   # Past Participle
```

Comment:
Going off of what we'd do with the `past participle`, i.e. when formatting, to simply use the `past participle` to construct the different conjugations for `* perfect` tenses.. 

I can't remember if we discussed the same for the `present participle` to construct the different conjugations for `* perfect continuous`, e.g. `have been walking`. Is that something that we do/will do? Do we need to grab the `present participle` as well?

Reply:
We haven't been doing continuous tenses until now, which is a good thought that maybe we do need this 🤔 I'd argue against present continuous, but maybe there's a use for perfect continuous :) I'm going to merge so we can play around with this a bit more, but have added a note for the weekly 😊